### DISCUSSION TITLE
cs 1.6 server erstellen

### DISCUSSION THREAD

**Discussion by rudi18711** [GameServerManagers/LinuxGSM#4120] [Q&A]
![Screenshot (48)](https://user-images.githubusercontent.com/125946304/220251942-3d0afc4e-4335-4f88-9a74-af0f67f1de2d.png)

I tried the following wanted to open a cs 1.6 server, the ports are also released for it but somehow it doesn't work with the instructions I found on steam can someone help me?

this is that what i would to try but its not working.... : https://steamcommunity.com/sharedfiles/filedetails/?id=710511353

**Comment by rudi18711** [comment]
i also used the tool which is here it also didn't work what can i do ?
https://github.com/Zeromix9/Server-Creation-Tool/releases

**Comment by rudi18711** [comment]
´
Will I get an answer again or what's going on and the one who sent me something this time I have windows and no LINUX!!!!

**Reply by Claiyc** [reply]
Why would you ask about that on the **Linux**GSM github then...?

That issue can have numerous causes. Disk space, some problem with the steamcmd cache, ...
steamcmd is quite hard to debug. If you're unfamiliar I'd suggest using a tool for server installation (eg. LinuxGSM lol). And before you repeat that you don't use linux.. Just do. You don't want to have a public gameserver running on your local computer anyway  (which I'm assuming you are trying to do) since it's a major security issue.

**Reply by rudi18711** [reply]
ok sorry by mistake so i understood correctly i should just start the linux program on my windows program =?

**Reply by Claiyc** [reply]
No not really.. I feel like you should first watch some yt-tutorials about the entire topic or just google around. It shows that you don't really know what you are talking about

**Reply by rudi18711** [reply]
watch out i'm in the discord on the server creation tool server there nobody can help me either i tried it with the help of my colleagues and tutorials is there a possibility to contact me on discord or so that you might help me a little can that would be super cool of you and i would show my gratitude in some way it's really a big request and i've been looking for it for a very long time there is the possibility that it would work that would be super nice of you

**Reply by rudi18711** [reply]
I did everything as in the tutorials and even made myself smarter. You would be my savior if you could help me

**Comment by dgibbs64** [comment]
This is a forum for LinuxGSM software and you are not using LinuxGSM.

**Reply by rudi18711** [reply]
oh sorry how do i get the windows help or the help i need =
